corehq/apps/hqwebapp/templates/hqwebapp/partials/pending_plan_notice.html

```diff
@@ -1,37 +1,36 @@
 {% load i18n %}
 
-<div class="plan-notice card mb-3 p-3">
+<div class="alert alert-primary">
+  <form
+    action="{% url 'dismiss_self_signup' domain %}"
+    method="POST"
+    class="float-end"
+  >
+    {% csrf_token %}
+    <button class="btn btn-close" type="submit">
+      <p class="visually-hidden">
```

Comment:
since we don't care about formatting, and since this is inside a `button` this `p` should be a `span`